src/prefect/serialization/schedule.py

```diff
@@ -39,6 +39,17 @@ class Meta:
     end_date = DateTimeTZ(allow_none=True)
     interval = fields.TimeDelta(precision="microseconds", required=True)
 
+    @post_dump
+    def _interval_validation(self, data: dict, **kwargs: Any) -> dict:
```

Comment:
Should this validation happen on the cloud side? What would happen if this was commented out and then a flow was registered with a sub 60 second clock?

Reply:
This is a great callout; I'd rather it happen prior to any cloud communication for efficiency purposes, but your comment does highlight that I should add an additional `post_load` check that users can't edit (because it'll run in Cloud)

Reply:
Updated